audpsychometric/__init__.py

```diff
@@ -2,11 +2,11 @@
 from audpsychometric.core import datasets
 from audpsychometric.core.datasets import list_datasets
 from audpsychometric.core.datasets import read_dataset
+from audpsychometric.core.gold_standard import agreement_categorical
```

Comment:
The module is called `gold_standard`. 
The functions have been renamed to all start with `agreement`. 

Should the module itself as well as the test file in tests then not be renamed accordingly?
`gold_standard` was initially chosen to be in order to confirm to the table names as published using `audb.publish`, and I agree that this is not the nicest way to term it.  
But I find that the term`agreement` for me points to a kind of coefficient that should sit together with the other reliabilities.  For me what happens in this module is the calculation of a kind of `consensus` rather than agreement calculation.

Reply:
Having the quote from @audeerington in mind:

> how much a rater aligns with the rater consensus across multiple samples, rather than how much (dis)agreement there is for a specific sample.

I would say, `mode()` is the only function in `gold_standard.py` that calculates a consensus at the moment, and the other functions calculate how much agreement we have for a specific sample, or how much agreement we have between raters.

Reply:
I will not be fussy about the exact names, but I found that there now is a kind of terminological armageddon:

```python
import audpsychometric

df = audpsychometric.read_dataset("wine")
df = df.pivot_table(index="Wine", columns="Judge", values="Scores")
print(df)
print("shape: ", df.shape)


agg = audpsychometric.agreement_categorical(df)
print("agreement_categorical", agg, agg.shape)

agg = audpsychometric.agreement_numerical(df, 0, 9)
print("agreement_numerical", agg, type(agg))

agg = audpsychometric.rater_agreement_pearson(df)
print("rater_agreement_pearson", agg, agg.shape)

agg = audpsychometric.mode(df)
print("mode value", agg, agg.shape)


agg = audpsychometric.evaluator_weighted_estimator(df)
print("ewe", agg, agg.shape)
```

This outputs
```
Judge    A    B    C    D
Wine                     
1      1.0  2.0  0.0  1.0
2      1.0  3.0  3.0  2.0
3      3.0  8.0  1.0  4.0
4      6.0  4.0  3.0  3.0
5      6.0  5.0  5.0  6.0
6      7.0  5.0  6.0  2.0
7      8.0  7.0  7.0  9.0
8      9.0  9.0  9.0  8.0
shape:  (8, 4)
agreement_categorical [0.5  0.5  0.25 0.5  0.5  0.25 0.5  0.75] (8,)
agreement_numerical 0.9037749551350623 <class 'float'>a
rater_agreement_pearson [0.85782072 0.7228383  0.83460774 0.8262462 ] (4,)
mode value [1 3 4 3 6 5 7 9] (8,)
ewe [0.96551936 2.21583295 3.85491628 4.01690183 5.51953114 5.02206086
 7.77442637 8.74510477] (8,)
```

I would have understood as a consensus the ones that have length 8 and are in the original metric. So the mean, the mode, the ewe .  I need to think a little bit more about it ( and collate some of the terms the psychologists use). This will take a little time.

Reply:
Good, that you spotted this:
```
agreement_numerical 0.9037749551350623 <class 'float'>a
```
this is a bug, and should have the same size as `agreement_categorical`.

Reply:
`rater_agreement_pearson` has another size as it returns a value per rater, whereas all other functions from above provide one value per stimulus.

Reply:
The error with `agreement_numerical()` is surprising as we have a test for the correct returned form:

https://github.com/audeering/audpsychometric/blob/f7296ee94d522646c67c742eb4ed6cdfdf833a7c/tests/test_gold_standard.py#L124-L130

Reply:
But I can confirm, that it returns the wrong shape:
```python
>>> audpsychometric.agreement_numerical([[1, 2, 3], [3, 4, 5]], 0, 10)
0.8367006838144548
```

Reply:
> But I can confirm, that it returns the wrong shape:
> 
> ```python
> >>> audpsychometric.agreement_numerical([[1, 2, 3], [3, 4, 5]], 0, 10)
> 0.8367006838144548
> ```

This is good in so far as it is then in the right place and should not go into reliability.